common.mk

```diff
@@ -0,0 +1,97 @@
+# This file should be included from chart-specific Makefiles
+# Those Makefiles should define the following values BEFORE including this file:
+#
+# CHART          # Chart name (must match directory name)
+# RELEASE_NAME   # Name of helm release
+# NAMESPACE      # Optional.  Namespace used when helming.  Default is 'lightbend'
+#
+
+#!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+#
+# NOTE:
+# Please be careful modifying this file.  Changes here will affect all of the 
+# helm-charts projects
+#
+#!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+ifndef CHART
+    $(error CHART is not set)
+endif
+
+ifndef RELEASE_NAME
+    $(error RELEASE_NAME is not set)
+endif
+
+# Default value for namespace used with helm.  Override if required.
+NAMESPACE ?= lightbend
+
+
+define banner
+	$(info === $@)
+endef
+
+# Note: We may want to require explicit lists.  As implemented, we may end
+# up with random cruft in the packages...
+COMPONENTS := $(wildcard */.)
+SUBCOMPONENTS := $(wildcard */*/.)
+
+HELM_CHARTS_DIR := $(abspath $(dir $(lastword $(MAKEFILE_LIST))))
```

Comment:
At glance this looks bad and order dependent... I can't confirm this works reliably in all cases by simply looking at it. Is it really necessary? If we assume this file is only ever included in subdirectory Makefiles, we know the root directory is always going to be `..`.